ci/build/release-github-assets.sh

```diff
@@ -10,7 +10,7 @@ main() {
   cd "$(dirname "$0")/../.."
   source ./ci/lib.sh
 
-  download_artifact release-packages ./release-packages
+  download_artifact release-packages ./release-packages "production" "v$VERSION"
```

Comment:
🧹 chore: `download_artifact` now takes 4 args. Although we set this as default fallback values when calling `download_artifact` I prefer to explicitly set them here so there's no guessing.